### PR TITLE
Fix formula

### PR DESCRIPTION
**Pull request by haslinghuis**
According to code it should be

image

But this one from @KarateBrot looks cleaner

image

### CODE REVIEW

**Review by nerdCopter — APPROVED** [review]

**Comment by nerdCopter** [comment]
approval for workflow, but formula still in discussion in discord.

**Review by nerdCopter — DISMISSED** [review]

**Comment by haslinghuis** [comment]
Do not merge yet. According to code it should be 

```bash
amps = ADC(mV) * 1000 / amperage_meter_scale + amperage_meter_offset
```

**Review by nerdCopter — APPROVED** [review]

File: docs/wiki/configurator/power-tab.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ Multiplier: 1 = this is to fine-tune the outcome of the calculation, a value of
 Settings for the amperage meter to measure the current draw correctly, using this formula:
 
 $$
-Current(Amps) = \frac\{ADC (mV)}\{amperage\_meter\_scale \times 10 + \frac\{amperage\_meter\_offset}\{1000}}
+Current(Amps) = (\frac{ADC (mV) * 10000}{amperage\_meter\_scale} + amperage\_meter\_offset) / 1000
```
Comment (KarateBrot) [inline docs/wiki/configurator/power-tab.md:76]: You can also write it like this
grafik
```suggestion
\text{Current}[A] = \frac{\text{ADC}[mV] \cdot 10}{\text{amperage_meter_scale}} + \frac{\text{amperage_meter_offset}}{1000}
```

Comment (KarateBrot) [reply docs/wiki/configurator/power-tab.md:76]: Or alternatively, like this
grafik
```latex
\text{Current}[A] = \left( \frac{\text{ADC}[mV] \cdot 10000}{\text{amperage_meter_scale}} + \text{amperage_meter_offset} \right) \cdot \frac{1}{1000}
```

Comment (haslinghuis) [reply docs/wiki/configurator/power-tab.md]: Thanks. All formula's including I'd proposed (which was based on code) have same outcome. But your first proposal looks the best option.

Comment (haslinghuis) [reply docs/wiki/configurator/power-tab.md]: Hmm first suggestion does not seem to be correct syntax.

Comment (KarateBrot) [reply docs/wiki/configurator/power-tab.md]: Both are equivalent

Comment (haslinghuis) [reply docs/wiki/configurator/power-tab.md]: Should be resolved in last commit

**Review by KarateBrot — APPROVED** [review]

File: docs/wiki/configurator/power-tab.md (hunk 1 above)
Comment (haslinghuis) [reply docs/wiki/configurator/power-tab.md]:
```
\text{Current}[A] = \frac{\text{ADC}[mV] \cdot 10}{\text{amperage_meter_scale}} + \frac{\text{amperage_meter_offset}}{1000}
```
vs
```
\text{Current[A] = } \frac{\text{ADC[mV]} \cdot 10}{\text{amperage\_meter\_scale}} + \frac{\text{amperage\_meter\_offset}}{1000}
```

**Pull request merged**